rules/typescript.js

```diff
@@ -18,7 +18,11 @@ module.exports = {
     '@typescript-eslint/no-unused-vars': 'error',
     '@typescript-eslint/no-useless-constructor': 'error',
     '@typescript-eslint/prefer-for-of': 'error',
-    '@typescript-eslint/prefer-optional-chain': 'error',
+    '@typescript-eslint/prefer-optional-chain': 'off',
+    "@typescript-eslint/explicit-module-boundary-types": "off",
+    "@typescript-eslint/ban-ts-comment": "off",
```

Comment:
I don't think this should be off, this should be a warning instead

Reply:
the same as above

Reply:
I personally believe you can't really prohibit the use the `@ts-ignore` comments with eslint rule. If someone is really desperate and want to silence TS they will use @ts-ignore and disable eslint rule for the line anyway. 
If we have a problem with @ts-ignore abuse IMO we should rather work on the culture where people really want to do type checking correctly and not abuse the @ts-ignore

Reply:
So IMO disabling the rule is legit. But I am fully open to other opinions.